apiserver/facades/client/charms/conversions.go

```diff
@@ -23,6 +24,7 @@ func convertOrigin(origin corecharm.Origin) params.CharmOrigin {
 	if origin.Channel != nil {
 		risk = string(origin.Channel.Risk)
 	}
+	chSeries, _ := series.VersionSeries(origin.Platform.Channel)
```

Comment:
added here since series is still needed for state (channel already validated but just in case....). will probably not need error check in 3.0 since series will not be needed